pelican/cache.py

```diff
@@ -1,15 +1,17 @@
+# -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
 import hashlib
 import logging
 import os
+
+from pelican.utils import mkdir_p
```

Comment:
I don't think this change is correct. From PEP-8:

> Imports should be grouped in the following order:
>
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
>
> You should put a blank line between each group of imports.

This is a local import and should be below the standard library import of pickle.